src/structures/Locale.js

```diff
@@ -0,0 +1,20 @@
+const Piece = require("./Piece");
+
+class Locale extends Piece {
+	constructor(client, store, file, options = {}) {
+		super(client, store, file, options);
+		this.name = options.name;
+		this.strings = options.strings ? options.strings : {};
+	}
+	t(key, ...args) {
```

Comment:
I don't think you should be using "t" here as the method along with "translate" because your not really translating anything. If anything, you could use get or fetch.

Reply:
I agree, msg.locale.get("key") would be better.